infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
+	Id            string   `json:"id"`
+	Title         string   `json:"title"`
+	Severity      string   `json:"severity"`
+	PublicId      string   `json:"publicId"`
+	Documentation string   `json:"documentation"`
```

Comment:
Is it "description" from the UI?

Reply:
this looks like the [documentation URL](https://github.com/snyk/snyk-intellij-plugin/blob/e20b59ccffac957d4b523351574c2b978d847362/src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt#L44), e.g. [SNYK-CC-K8S-13](https://snyk.io/security-rules/SNYK-CC-K8S-13)

I tried to keep the naming consistent with the intelliJ implementation, but if it doesn't make sense we could rename it to something more appropriate like `DocumentationURL`